multi-int-store/src/main/java/com/connexta/multiintstore/adaptors/S3StorageAdaptor.java

```diff
@@ -117,4 +117,9 @@ public RetrieveResponse retrieve(@NotBlank final String key) throws StorageExcep
       throw t;
     }
   }
+
+  @Override
+  public boolean objectExists(String key) {
+    return (amazonS3.doesObjectExist(bucket, key)) ? true : false;
```

Comment:
:exclamation:  `doesObjectExist` returns a boolean. No need for the ternary expression.

Reply:
Not sure why I didn't see this when I wrote it. 😂